src/components/AuthLayout/index.js

```diff
@@ -133,7 +231,7 @@ class AuthLayout extends Component {
             className={styles.form}
           >
             {Heading}
-            {toPreview}
+            {toPreview === undefined ? authPreview : toPreview}
```

Comment:
can be modified as `toPreview ? authPreview : toPreview`

Reply:
it should be `toPreview ? toPreview : authPreview`  i.e `toPreview || authPreview`
Fixed!